back/src/middlewares/authenticate.ts

```diff
@@ -31,6 +33,8 @@ const authenticate = async (
   req.account = {
     id: account.id,
     profileType: account.profile_type,
+    longitude: profile?.longitude,
```

Comment:
Maybe add `|| null` to prevent `undefined` values ?

Reply:
Yeah I modified it too